package.json

```diff
@@ -4,27 +4,25 @@
   "engines": {
     "node": "12.4.0"
   },
-  "description": "Form Builder Editor app (Node version)",
+  "description": "Form Builder Editor (for Node)",
   "main": "index.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/ministryofjustice/fb-editor-node.git"
   },
   "scripts": {
     "start": "node bin/start.js",
-    "lint": "npm-run-all lint:lib",
-    "lint:lib": "eslint lib",
-    "test": "echo \"No tests specified yet\" && exit 0",
-    "test:all": "npm-run-all test:unit",
-    "test:unit": "tape lib/**/*.unit.spec.js"
+    "lint": "eslint lib",
+    "lint:fix": "eslint lib --fix",
+    "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "author": "github@alex.fu2k.org",
+  "author": "Form Builder Team <form-builder-team@digital.justice.gov.uk>",
   "license": "MIT",
   "devDependencies": {
-    "@ministryofjustice/eslint-config-fb": "^1.0.7",
-    "tape": "^4.11.0"
+    "@ministryofjustice/eslint-config-fb": "^1.1.2"
   },
   "dependencies": {
+    "@ministryofjustice/fb-components-core": "^3.1.0-7",
```

Comment:
this is outdated now i believe. runner now loads the library so this this line is not needed. can you rebase with master please to make sure

Reply:
Good spot

Reply:
Line gone